tests/vexflow_test_helpers.ts

```diff
@@ -208,17 +208,18 @@ export class VexFlowTests {
 
   /**
    * Each font stack is a prioritized list of font names.
+   * Music engraving fonts are paired with their recommended text fonts (if specified by the designer).
    */
   static FONT_STACKS: Record<string, string[]> = {
-    Bravura: ['Bravura', 'Roboto Slab'],
-    Gonville: ['GonvilleSmufl', 'Bravura', 'Roboto Slab'],
-    Petaluma: ['Petaluma', 'Bravura', 'Petaluma Script'],
-    Leland: ['Leland', 'Bravura', 'Roboto Slab'],
-    MuseJazz: ['MuseJazz', 'Bravura', 'Academico'],
-    Gootville: ['Gootville', 'Bravura', 'Academico'],
-    'Finale Ash': ['Finale Ash', 'Bravura', 'Academico'],
-    'Finale Broadway': ['Finale Broadway', 'Bravura', 'Academico'],
-    'Finale Maestro': ['Finale Maestro', 'Bravura', 'Academico'],
+    Bravura: ['Bravura', 'Academico'],
+    'Finale Ash': ['Finale Ash', 'Finale Ash Text'],
+    'Finale Broadway': ['Finale Broadway', 'Finale Broadway Text'],
+    'Finale Maestro': ['Finale Maestro', 'Finale Maestro Text'],
+    Gonville: ['GonvilleSmufl', 'Academico'],
+    Gootville: ['Gootville', 'Edwin'],
+    Leland: ['Leland', 'Edwin'],
+    MuseJazz: ['MuseJazz', 'MuseJazz Text'],
+    Petaluma: ['Petaluma', 'Petaluma Script'],
```

Comment:
Should we add Sebastian and Leipzig?

Reply:
Sure, I can add them. How long does flow.html tests take on your computer? It is pretty slow on my laptop. At some point we will need to figure out an easy way to run only a subset of the tests / subset of the fonts.

Reply:
I agree that we should probably have the opportunity to select the fonts when testing.

Reply:
OK we leave it out for now.